ruma-client-api/src/r0/keys.rs

```diff
@@ -49,7 +49,7 @@ pub struct CrossSigningKey {
     pub keys: BTreeMap<String, String>,
 
     /// Signatures of the key. Only optional for master key.
-    #[serde(skip_serializing_if = "BTreeMap::is_empty")]
+    #[serde(skip_serializing_if = "BTreeMap::is_empty", default = "BTreeMap::new")]
```

Comment:
I think the default comes first everywhere else, also since `BTreeMap::new` is the same as `<BTreeMap<_, _> as Default>::default` this can just be

```suggestion
    #[serde(default, skip_serializing_if = "BTreeMap::is_empty")]
```